app/src/RoomClient.js

```diff
@@ -680,14 +680,7 @@ export default class RoomClient
 	{
 		logger.debug('receiveFromChildWindow() | [data:"%o"]', data);
 
-		const { picture } = data;
-
-		let displayName;
-
-		if (typeof(data.displayName) === 'undefined' || !data.displayName)
-			displayName = '';
-		else
-			displayName = data.displayName;
+		const { displayName, picture } = data;
```

Comment:
Why do you remove the sanity check? I'm not sure what this is about can you explain?